pkg/apis/pipeline/v1beta1/workspace_types.go

```diff
@@ -77,6 +77,9 @@ type WorkspaceBinding struct {
 	// Secret represents a secret that should populate this workspace.
 	// +optional
 	Secret *corev1.SecretVolumeSource `json:"secret,omitempty"`
+	// CSI (Container Storage Interface) represents ephemeral storage that is handled by certain external CSI drivers (Beta feature).
```

Comment:
I'm assuming "beta feature" is copied from k8s docs? we will likely want to introduce this as an alpha feature

Reply:
Thanks! Here's an [example](https://github.com/tektoncd/pipeline/blob/main/pkg/apis/pipeline/v1beta1/task_validation.go#L143) of guarding something behind an alpha feature flag.